src/App.jsx

```diff
@@ -14,70 +15,83 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {getHeader(selectedGood, setSelectedGood)}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              className={
+                  good === selectedGood
+                    ? 'has-background-success-light'
+                    : ''
+                  }
             >
```

Comment:
Add key attribute for tr
```suggestion
            <tr
              key={good}
              data-cy="Good"
              className={
                  good === selectedGood
                    ? 'has-background-success-light'
                    : ''
                  }
            >
```